src/ensembl/src/content/app/help/Help.tsx

```diff
@@ -49,12 +53,23 @@ const Help = () => {
     endpoint: `/api/docs/article?url=${encodeURIComponent(location.pathname)}`,
     skip: isIndexPage
   });
+
+  let breadcrumbs: string[] | undefined = [];
+  if (!isIndexPage && menu && article) {
+    breadcrumbs = buildBreadcrumbs(menu, article);
+  }
+
   const main = isIndexPage ? (
     <main className={styles.main}>
-      <HelpLanding />
+      <div className={styles.breadcrumbsContainer}>
+        <Breadcrumbs breadcrumbs={['Overview']} />
+      </div>
+      <div className={styles.articleContainer}>
+        <HelpLanding />
+      </div>
     </main>
-  ) : article ? (
-    <MainContent article={article} />
+  ) : article && breadcrumbs ? (
```

Comment:
`&& breadcrumbs` is always true if it's initialised with an empty array.

Reply:
Please refer to the comment above

Reply:
I'm confused. Now `breadcrumbs` is definitely always truthy. Shouldn't you remove them from the check?

Reply:
I've removed it 👍